projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.ts

```diff
@@ -36,11 +37,16 @@ export class DocumentDetailViewComponent implements DetailRecord, OnInit {
   /** Document record */
   record: any;
 
+  /** Css class for dd in template */
+  ddCssClass = 'col-sm-6 col-md-8 mb-0';
```

Comment:
Why don't put this into a CSS file ? In my sense, it will be a best practice

Reply:
@jma suggestion